airflow/providers/airbyte/__init__.py

```diff
@@ -29,10 +29,7 @@
 
 __version__ = "3.8.0"
 
-try:
```

Comment:
Ups. I see it's been done :)

Reply:
I did, it's the very last file :)

Reply:
> BTW. It could be done by changing template and running breeze release-management prepare-provider-documentation --reapply-templates-only

Oh, that would have been faster. Next time I'll keep that in mind.